pinc/ProjectSearchResults.inc

```diff
@@ -298,7 +298,7 @@ class StateColumn extends Column
         }
     }
 
-    private function _echo_project_state_option($project_state, $selected, $disabled = 0, $reason = '')
+    private function _echo_project_state_option(string $project_state, bool $selected, bool $disabled = false, $reason = ''): void
```

Comment:
```suggestion
    private function _echo_project_state_option(string $project_state, bool $selected, bool $disabled = false, string $reason = ''): void
```
`$transition->why_disabled()` is typed to return a string and that's the only thing we pass in here besides the default so might as well type it.